app/src/main/res/values/seed_phrase_strings.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
+    <!--Derivable Accounts-->
+    <string name="derivable_accounts_title">Derivable Accounts</string>
+    <string name="derivation_path_title">Derivation path</string>
+    <string name="derivation_path_info">This is the thing you use to get all your accounts from your mnemonic phrase. By default, P2P wallet will use m/44\'/501\'/0\'/0\' as the derivation path for the main wallet.</string>
+    <string name="derivation_continue_button">Continue</string>
```

Comment:
Why do we create a separate string file for each screen?
When we'll have 50, 100 screens there will be too much files like that.
I offer to split by feature at least